packages/squiggle-lang/__tests__/Distributions/Dotwise_test.res

```diff
@@ -0,0 +1,50 @@
+open Jest
+open Expect
+open TestHelpers
+open FastCheck
+open Arbitrary
+open Property.Sync
+
+describe("dotSubtract", () => {
+  test("mean of normal minus exponential (unit)", () => {
```

Comment:
testing the mean of this is unintuitive to me. A pointwise manipulation wouldn't even return a normalized dist, for one thing.

Reply:
It's something I observed in playground, but it didn't work when the normal had non-1 stdevs. Not sure if it's a kind of invariant that we want to expect of the algebra. 

Maybe a better test would be that `mean(a - b) = - mean(b - a)`

Reply:
``mean(a - b) = - mean(b - a)`` -> This is a pointwise operation, not a regular operation. So the mean() feels pretty weird to me.